dev-python/pycryptodome/pycryptodomex-3.16.0.recipe

```diff
@@ -0,0 +1,88 @@
+SUMMARY="Cryptographic library for Python"
+DESCRIPTION="PyCryptodome is a self-contained Python package of low-level cryptographic \
+primitives.
+
+PyCryptodome is a fork of PyCrypto. It brings several enhancements with respect \
+to the last official version of PyCrypto (2.6.1).
+"
+HOMEPAGE="https://www.pycryptodome.org/"
+LICENSE="BSD (2-clause)
+	Public Domain
+	Unlicense
+	"
+COPYRIGHT="2014-2022 Legrandin"
+REVISION="1"
+SOURCE_URI="https://pypi.python.org/packages/source/p/pycryptodome/pycryptodome-$portVersion.tar.gz"
+CHECKSUM_SHA256="0e45d2d852a66ecfb904f090c3f87dc0dfb89a499570abad8590f10d9cffb350"
+PATCHES="pycryptodomex-$portVersion.patchset"
+
```

Comment:
Needs: SOURCE_DIR="pycryptodome-$portVersion"
I would rather see the two of them put together in one recipe. :)